components/constants/pref_names.h

```diff
@@ -52,7 +52,8 @@ extern const char kNewTabPageShowGemini[];
 extern const char kNewTabPageShowBraveTalk[];
 extern const char kNewTabPageHideAllWidgets[];
 extern const char kNewTabPageShowsOptions[];
-extern const char kNewTabPageCustomBackgroundEnabled[];
```

Comment:
One more thing - Let's cleanup this. we introduced `components/constatns` temporarily for fixing layer violation.
We will move prefs in this file to appropriated components. I think these two prefs can be in `//brave/browser/ntp_xxx` ?

Reply:
Thanks! done!